src/components/organelle-wrapper.tsx

```diff
@@ -152,6 +150,12 @@ class OrganelleWrapper extends React.Component<OrganelleWrapperProps, OrganelleW
       this.model = m;
       this.completeLoad();
     });
+
+    autorun(() => {
```

Comment:
Previously, some state changes were monitored in `componentWillReceiveProps`. Since I'm now passing less data through the props, it seemed like the right thing to do was monitor state in an `autorun` function. This has the downside of not showing `prevState`, but I could store that separately if I need to.

Does this seem like a correct way to transition from React to MobX?

Reply:
For side effects other than rendering (like changing the component state as is done here) I think an autorun makes sense.